openapi-v3.yaml

```diff
@@ -223,6 +247,9 @@ paths:
           name: order_by
           schema:
             type: string
+            anyOf:
+            - enum: [id, created_at, -id, -created_at]
+            - type: "string"
```

Comment:
I think this is redundant with the enum - at least redoc displays it as two separate options:

![Screen Shot 2022-06-03 at 11 54 01 AM](https://user-images.githubusercontent.com/200728/171901492-6b6fd12d-fe75-4a1e-94af-695f14ba6228.png)

Reply:
I think this is the most correct thing though (or just `string` with examples) - I missed your comment earlier about there being other options and these just being the most common

Reply:
I'm going to remove this for now since it's an incomplete list anyways.